docs/user/dql/functions.rst

```diff
@@ -2331,6 +2331,31 @@ Example::
     +---------------------+---------------------+
 
 
+POSITION
+------
+
+Description
+>>>>>>>>>>>
+
+Usage: The syntax POSITION(substr IN str) returns the position of the first occurrence of substring substr in string str. Returns 0 if substr is not in str. Returns NULL if any argument is NULL.
+
+Argument type: STRING, STRING, INTEGER
+
+Return type integer:
+
+(STRING IN STRING) -> INTEGER
+
+Example::
+
+    os> SELECT POSITION('world' IN 'helloworld')
```

Comment:
can you add the 0 case, like:
```
POSITION('invalid' IN 'helloworld')
```

Reply:
Addressed in 9a6b7b5d8962cff8af410638da5151e9ee6a36b9